script.skinshortcuts/default.py

```diff
@@ -307,11 +304,22 @@ def _manage_shortcuts( self, group, defaultGroup, nolabels, groupname ):
         if homeWindow.getProperty( "skinshortcuts-loading" ) and int( calendar.timegm( gmtime() ) ) - int( homeWindow.getProperty( "skinshortcuts-loading" ) ) <= 5:
             return
 
+        # Get current Super Favourites context menu status
+        sfMenu = None
+        if xbmc.getCondVisibility( "System.HasAddon(plugin.program.super.favourites)" ):
```

Comment:
From a UX perspective, explaining to the user why I want to disable something that can't be used and that they won't notice isn't enabled and asking for their permission to do so - in the three lines we're given and with the default option being 'No' - isn't something I consider acceptable from a UX perspective.

There are alternatives (such as editing the keymap so that the Skin Shortcuts window isn't affected when the SF global context menu is active) that I can look into, but can I get confirmation that this is considered to be blocking the update on the repo before I do?